wicket-devutils/src/main/java/org/apache/wicket/devutils/debugbar/InspectorDebugPanel.java

```diff
@@ -73,7 +74,22 @@ protected ResourceReference getImageResourceReference()
 	@Override
 	protected IModel<String> getDataModel()
 	{
-		return new Model<>("Inspector");
+		return new IModel<String>()
+		{
+			private static final long serialVersionUID = 1L;
+
+			@Override
+			public String getObject()
+			{
+				Page enclosingPage = getPage();
+				long pageSize = WicketObjects.sizeof(enclosingPage);
+				Bytes pageSizeInBytes = (pageSize > -1 ? Bytes.bytes(pageSize) : null);
+				String pageSizeAsString = pageSizeInBytes != null ? pageSizeInBytes.toString()
+					: "unknown";
+
+				return "Page: " + pageSizeAsString;
```

Comment:
`Page size: ` ? Otherwise it may render `Page: unknown` and it will be a little confusing

Reply:
Actually I think this was better before where we just showed a "Inspector" label - serializing the containing page might not something we want to do on all pages. I'll revert this change.